next-docs/public/examples/bannerTW/Singleline.tsx

```diff
@@ -0,0 +1,16 @@
+import { Banner, Button } from '@heathmont/moon-core-tw';
+
+const Example = () => {
+  const link = (
+    <Button variant="ghost">Resend verification link to email ›</Button>
+  );
+  return (
+    <Banner
+      description="Your email is not verified. Verify to enable withdrawals."
+      link={link}
+      isMultiline={false}
```

Comment:
Was it confirmed that Multiline view is default? If yes, please revert the logic: 
`isSingline={true}`
We turn on a new view from default. 
Or we can discuss this if you want.

Reply:
@yarema184  again, I used current state of Banner component as a refernce. Should I do that or look somewhere else for a reference when building TW components? Because currently default state is multi line as you can see..
![33](https://user-images.githubusercontent.com/58221984/177655668-545ea073-23d5-49c3-b0c2-660a1003a76c.jpg)
.